### DISCUSSION TITLE
Welcome to AutoIntegrate Discussions!

### DISCUSSION THREAD

**Discussion by jarmoruuth** [jarmoruuth/AutoIntegrate#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Miflur** [comment]
Hello everyone,
I've been using Pixinsigth for a few years and I've discovered AutoIntegrate that was doing me a good service because I don't have much time for treatment. Since version V1.71 it no longer wants to work. I use the latest update of Pix.
If anyone can help me. Thank you in advance.

**Reply by jarmoruuth** [reply]
Hi! What error do you get?

**Comment by Miflur** [comment]
Hi, I solved it. I needed the latest version of Pixinsight. Now it works well.

Thank you very much for your message.
Best regards

**Reply by jarmoruuth** [reply]
Ok! Great to hear that problem was resolved. I know at least one problem where I used a function that is not available except in the last PixInsight version. I will fix that for the next version.
